packetfabric/manifest.json

```
@@ -0,0 +1,63 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b8e02be2-3d0b-4dd2-a014-f78fdab1710a",
+  "app_id": "da10a120-217b-40f3-8b7f-7dc2fdea3b94",
```

```suggestion
  "app_id": "packetfabric",
```
This should be a more human-readable id name

---

This is the app_id given to me on clicking kickoff publishing process and the instruction says to use this as app_id in the manifest

---

Oh I see. Sorry for the confusion -- that `uuid` from the publishing process should actually replace the `app_uuid` field in the manifest.

---

We will update the copy to be more clear, thank you for calling that out.